web/gen-header

```diff
@@ -3,75 +3,49 @@
 #
 # Generates the top site navigation part of the HTML page.
 # Argument must be one of the site parts.
-#
 
 import datetime
 import locale
-import oplcsv
 import sys
+sys.path.append('../scripts')
+import oplcsv
 
 
 def showdbstats():
-    locale.setlocale(locale.LC_ALL, 'en_US')
+    locale.setlocale(locale.LC_ALL, 'en_US.utf8')
     csv = oplcsv.Csv('../build/openpowerlifting.csv')
     meets = oplcsv.Csv('../build/meets.csv')
 
     num_entries = locale.format('%d', len(csv.rows), grouping=True)
     num_meets = locale.format('%d', len(meets.rows), grouping=True)
     compiledate = datetime.datetime.now().strftime("%Y-%m-%d")
 
-    print(' (%s Entries; %s Meets; Updated %s)' % (num_entries, num_meets, compiledate))
-
+    print('%s Entries; %s Meets; Updated %s' % (num_entries, num_meets, compiledate))
 
-def main(tabname):
+def addnav(activetab, tab, title):
+    active = 'nav__link--active' if activetab == tab else ''
+    print('<a href="{0}.html" class="nav__link {1}">{2}</a>'.format(tab, active, title))
 
+def main(activetab):
+    assert activetab in ['rankings', 'lifters', 'meet', 'meetlist', 'data', 'contact']
     print('<div id="header">')
-    print('OpenPowerlifting')
-
     print('<div class="nav">')
+    print('<a href="index.html" class="nav__link brand">OpenPowerlifting</a>')
```

Comment:
The CSS style of the `OpenPowerlifting` text changed somewhere, and it's much bigger than before.